site/components/Navigation/index.tsx

```diff
@@ -58,47 +72,360 @@ export const Navigation: FC<Props> = ({
+                    id: "shared.stake",
+                  })}
+                  key="stake"
+                  url={createLinkWithLocaleQuery(urls.stake, locale)}
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "BOND KLIMA",
+                    id: "shared.bond",
+                  })}
+                  key="bond"
+                  url={createLinkWithLocaleQuery(urls.bonds, locale)}
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "WRAP SKLIMA",
+                    id: "shared.wrap",
+                  })}
+                  key="wrap"
+                  url={createLinkWithLocaleQuery(urls.wrap, locale)}
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "OFFSET",
+                    id: "shared.offset",
+                  })}
+                  key="offset"
+                  url={createLinkWithLocaleQuery(urls.offset, locale)}
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "INFO",
+                    id: "shared.info",
+                  })}
+                  key="info"
+                  url={createLinkWithLocaleQuery(urls.info, locale)}
+                />,
+              ]}
+            />
+            <NavItemMobile
+              name={t({ message: "Infinity", id: "shared.infinity" })}
+              toggledNavItemID={toggledNavItemID}
+              setToggledNavItemID={setToggledNavItemID}
+              url="/infinity"
+              id="Infinity"
+              // subMenu={
+              //   [
+              // <NavItemMobile
+              //   name={t({
+              //     message: "INTRODUCTION",
+              //     id: "shared.intoduction",
+              //   })}
+              //   key="intro"
+              //   url="/infinity"
+              //   active={activePage === "Infinity"}
+              // />,
+              // <NavItemMobile
+              //   name={t({
+              //     message: "HOW TO PLEDGE",
+              //     id: "shared.how_to_pledge",
+              //   })}
+              //   key="how pledge"
+              //   // not sure about this url
+              //   url="/blog"
+              // />,
+              // <NavItemMobile
+              //   name={t({
+              //     message: "LEADER BOARD",
+              //     id: "shared.leader_board",
+              //   })}
+              //   key="leader board"
+              //   url="/blog"
+              // />,
+              //   ]
+              // }
+            />
+            <NavItemMobile
+              name={t({ message: "Love Letters", id: "shared.loveletters" })}
+              toggledNavItemID={toggledNavItemID}
+              setToggledNavItemID={setToggledNavItemID}
+              id="Love Letters"
+              url={createLinkWithLocaleQuery(urls.loveletter, locale)}
```

Comment:
loveletter doesn't support translation unfortunately
```suggestion
              url={urls.loveletter}
```

Reply:
fixed